doc/builtin_models.rst

```diff
@@ -312,17 +320,17 @@ could to define this in a script:
 
     script = """
     def mycurve(x, amp, cen, sig):
-        loren = lorentzian(x, amplitude=amp, center=cen, sigma=sig)
-        gauss = gaussian(x, amplitude=amp, center=cen, sigma=sig)
-        return log(loren) * gradient(gauss) / gradient(x)
+	loren = lorentzian(x, amplitude=amp, center=cen, sigma=sig)
+	gauss = gaussian(x, amplitude=amp, center=cen, sigma=sig)
+	return log(loren) * gradient(gauss) / gradient(x)
```

Comment:
are you sure this will render correctly indented Python code on the documentation website (here and in other places where you made similar whitespace changes) so that, for example, people can copy-and-paste code from the docs to their Python shell?

Reply:
not sure what is happening there... My editor seems to be putting tabs into rst files for indents.  They do seem to render to HTML correctly